### PR TITLE
Send SDK override and version to FFI

### PR DESCRIPTION
**Pull request by bcherry**
This adds the new required params for SDK and SDK version, as added in https://github.com/livekit/rust-sdks/pull/471

It looks like this works but it's stuck on a different protobuf error during connection so I haven't verified they do make it through unharmed to the SFU but I believe it will work.

### CODE REVIEW

File: livekit-rtc/livekit/rtc/_ffi_client.py [hunk 1]
```diff
@@ -54,14 +55,20 @@ def get_ffi_lib():
     res = importlib.resources.files("livekit.rtc.resources") / libname
     ctx = importlib.resources.as_file(res)
     path = _resource_files.enter_context(ctx)
+    print(path)
```
Comment (theomonnom) [inline livekit-rtc/livekit/rtc/_ffi_client.py:58]:
```suggestion
```

File: livekit-rtc/livekit/rtc/_ffi_client.py [hunk 2]
```diff
@@ -204,8 +211,13 @@ def instance(cls) -> "FfiClient":
...
         self._lock = threading.RLock()
         self._queue = FfiQueue[proto_ffi.FfiEvent]()
-
-        ffi_lib.livekit_ffi_initialize(ffi_event_callback, True)
+        
+        ffi_lib.livekit_ffi_initialize(
+            ffi_event_callback,
+            True,
+            b"python",
+            version("livekit").encode('utf-8')
```
Comment (theomonnom) [inline livekit-rtc/livekit/rtc/_ffi_client.py:219]: let's use our version.py as the ground truth 
from ..version import __version__

**Review by theomonnom — APPROVED** [review]
otherwise lgtm

File: livekit-rtc/livekit/rtc/_ffi_client.py [hunk 3]
```diff
@@ -205,7 +211,9 @@ def __init__(self) -> None:
         self._lock = threading.RLock()
         self._queue = FfiQueue[proto_ffi.FfiEvent]()
 
-        ffi_lib.livekit_ffi_initialize(ffi_event_callback, True)
+        ffi_lib.livekit_ffi_initialize(
+            ffi_event_callback, True, b"python", version("livekit").encode("utf-8")
```
Comment (theomonnom) [inline livekit-rtc/livekit/rtc/_ffi_client.py:215]: encode should be ascii

File: livekit-rtc/livekit/rtc/_ffi_client.py (hunk 2 above)
Comment (bcherry) [reply livekit-rtc/livekit/rtc/_ffi_client.py]: ah - thank you! didn't see that

File: livekit-rtc/livekit/rtc/_ffi_client.py (hunk 1 above)
Comment (bcherry) [reply livekit-rtc/livekit/rtc/_ffi_client.py]: oops, fixed

File: livekit-rtc/livekit/rtc/_ffi_client.py (hunk 3 above)
Comment (bcherry) [reply livekit-rtc/livekit/rtc/_ffi_client.py]: makes sense - updated

**Pull request merged**